Code/Legacy/CryCommon/platform_impl.cpp

```diff
@@ -404,7 +404,7 @@ _MS_ALIGN(64) uint32  BoxSides[0x40 * 8] = {
     0, 0, 0, 0, 0, 0, 0, 0, //3d
     0, 0, 0, 0, 0, 0, 0, 0, //3e
     0, 0, 0, 0, 0, 0, 0, 0, //3f
-};
+} _ALIGN(64);
```

Comment:
`alignas(64)` can be used on this variable

Reply:
same here